src/Fay/Types/Js.hs

```diff
@@ -16,6 +16,7 @@ import           Language.Haskell.Exts
 -- | Statement type.
 data JsStmt
   = JsVar JsName JsExp
+  | JsDecl JsName JsExp
```

Comment:
Maybe it would be better to rename this to `JsMapVar` or similar to reflect the TS type, reuse it for JS-generation as well, and then only have the printer distinguish between js & ts output?

Reply:
Or maybe just add a Bool to JsVar to distinguish between the two?